src/invoice_utils/engine/_engine.py

```diff
@@ -33,6 +25,17 @@ def __init_invoice(self):
             "totals": {"price": 0, "total": 0, "extra": {}},
         }
 
+    @staticmethod
+    def __set_default_rules():
+        fpath = pathlib.Path(__file__).parent.parent / "basic.json"
+        if not fpath.exists():
+            raise InvoicingInputError("basic.json")
```

Comment:
In the case of `basic.json`, the web app shouldn't start up if this file is missing or invalid. That check belongs in the entry point of the app or in the app settings.